canonical-web-and-design/charmhub.io
# Proposing a clean up of the filter component

## Done
While looking into the checkbox jumping issue I noticed there are a lot of bespoke styling in the filters section of the site so I would like to propose this clean up of the filter component to use Vanilla much more so we don't get more difficult issues like this.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045
- Check that the filter section looks good
- Check it works as before
